Jenkinsfile

```diff
@@ -45,3 +41,42 @@ def set_properties(){
     ])
   ])
 }
+
+def prBuilder(){
+  def mvnArgs="-Pdeb -U -Phadoop-2.7 -Dhadoop.version=2.8.2 -Pkinesis-asl -Pyarn -Phive -Phive-thriftserver -Dpyspark -Dsparkr"
+  stage('PR Build'){
+    sh "build/mvn ${mvnArgs} -DskipTests clean install"
+  }
+  stage('PR Test'){
+    sh "build/mvn ${mvnArgs} install"
+  }
+}
+
+def releaseBuilder(){
+  def mvnArgs="-Pdeb -U -Phadoop-2.7 -Dhadoop.version=2.8.2 -Pkinesis-asl -Pyarn -Phive -Phive-thriftserver -Dpyspark -Dsparkr -DskipTests -Dgpg.skip=true -Dmaven.javadoc.skip=true"
+  stage('Release Clean'){
+    sh "build/mvn ${mvnArgs} release:clean"
+  }
+  stage('Release Prepare'){
+    sh "build/mvn ${mvnArgs} release:prepare"
```

Comment:
Pretty sure you are also still going to need `-Darguments="$args"` for both the Release Prepare and Release Perform stages.